src/Components/CommentSection.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FormEvent, useState} from "react";
+import CommentDialogSection from "./CommentDialogSection";
+import {addComment, getComments} from "../Service/User/RequestsForUsers";
+import {useAppSelector} from "../Hooks/hook";
+import {commentSchema} from "../validator";
+import {ToastNotifyError, ToastNotifySuccess} from "../Helpers";
+import {CommentFromDB, User} from "../types/typeSection";
+import CoreButton from "./common/CoreButton";
+
+interface Props {
+    postId: number
+}
+
+const CommentSection: React.FC<Props> = ({postId}) => {
+    const user:User|null = useAppSelector(state => state.auth.auth)
+    const [loading,setLoading] = useState<boolean>(false)
+    const [open, setOpen] = useState<boolean>(false);
+    const [comment, setComment] = useState<string>('')
+    const token: string = useAppSelector(state => state.auth.token);
+    const [spinIsActive, setSpinIsActive] = useState<boolean>(false)
+    const [postComments, setPostComments] = useState<CommentFromDB[]>([])
+    const handleClickOpen = async (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.preventDefault()
+        console.log(postId)
+            setSpinIsActive(true)
+            const getPostComments = await getComments(postId, token)
+            setPostComments([...getPostComments])
+            setSpinIsActive(false)
+            setOpen(true);
+    };
+    const handleCommentAdding = async (e: FormEvent): Promise<void> => {
+        e.preventDefault()
+        if(user){
+            const validationRules: boolean = await commentSchema.isValid({comment})
```

Comment:
Huge mistake, doing validation on this stage, research and find best approach to make validation

Reply:
it's changed now to formik